Added two more image generative models (#53)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

The exact procedure how DALL-E models work proposed by Ramesh et. al is explained in their paper. \cite{ramesh2021zeroshot}

> Another image generative model is the \textbf{midjourney} \cite{midjourney}. This model was created by an independent research lab with the same name. It can synthesize images from textual descriptions. Midjourney tends to generate surrealistic images and is popular among artists and it uses modern deep learning methods to create high-quality images.

Reviewer: ```suggestion
Another image generative model is \textbf{midjourney} \cite{midjourney}. This model was created by an identically named and independent research lab. It can synthesize images from textual descriptions. Midjourney tends to generate surrealistic images and is popular among artists. It uses modern deep learning methods to create high-quality images.
```

> \textbf{Imagen} is a text-to-image diffusion model with an unprecedented degree of photorealism and a deep level of language understanding. Imagen builds on the power of large transformer language models in understanding text and hinges on the strength of diffusion models in high-fidelity image generation. More information can be found either on their website \cite{imagen} or in their paper \cite{saharia2022photorealistic}. 

Reviewer: I would not write more information can be found ... This does imho not fit into a scientific report.
```suggestion
\textbf{Imagen} is a text-to-image diffusion model with an unprecedented degree of photorealism and a deep level of language understanding. Imagen builds on the power of large transformer language models in understanding text and hinges on the strength of diffusion models in high-fidelity image generation. \cite{imagen} \cite{saharia2022photorealistic}. 
```

Reviewer 2: Maybe instead of writing, that more information can be found there, just write another sentence explaining the procedure of the model image generating process or just as Max said, delete the sentence

Author: I can delete the sentence but in my opinion it is good enough to write what already there and if someone wants to know more, there are links where they can read everything about the certain models.

Reviewer 3: referencing a paper or website link, as you did is enough and everyone will understand that they can find more information useing them imo. but for me it doesnt matter that much if you want to write it you might as well do that.
